packages/mco/components/create-dr-policy/select-cluster-list.tsx

```diff
@@ -1,204 +1,177 @@
+import { getPageRange, referenceForModel } from '@odf/shared/utils';
+import {
+  GreenCheckCircleIcon,
+  ListPageFilter,
+  RedExclamationCircleIcon,
+  StatusIconAndText,
+  useK8sWatchResource,
+  useListPageFilter,
+} from '@openshift-console/dynamic-plugin-sdk';
 import {
-  DataList,
-  DataListItem,
-  DataListItemRow,
-  DataListCheck,
-  DataListItemCells,
-  DataListCell,
-  Toolbar,
-  ToolbarContent,
-  ToolbarItem,
-  SearchInput,
-  DataListCheckProps,
-  TextContent,
-  TextVariants,
+  Grid,
+  GridItem,
+  Pagination,
+  PaginationVariant,
   Text,
-  Tooltip,
 } from '@patternfly/react-core';
+import { Td } from '@patternfly/react-table';
 import {
-  MAX_ALLOWED_CLUSTERS,
-  MANAGED_CLUSTER_REGION_CLAIM,
-  MANAGED_CLUSTER_JOINED,
-  MANAGED_CLUSTER_CONDITION_AVAILABLE,
   MCO_CREATED_BY_LABEL_KEY,
   MCO_CREATED_BY_MC_CONTROLLER,
 } from '../../constants';
 import { ACMManagedClusterViewModel } from '../../models';
 import { ACMManagedClusterKind, ACMManagedClusterViewKind } from '../../types';
+import {
+  COLUMN_NAMES,
+  COUNT_PER_PAGE_NUMBER,
+  getColumnHelper,
+  getColumns,
+  getManagedClusterInfoTypes,
+  INITIAL_PAGE_NUMBER,
+  isRowSelectable,
+} from './utils/cluster-list-utils';
 import {
   DRPolicyAction,
   DRPolicyActionType,
   ManagedClusterInfoType,
-  ODFConfigInfoType,
-} from './reducer';
-import './select-cluster-list.scss';
+} from './utils/reducer';
 
-const getFilteredClusters = (
-  clusters: ManagedClusterInfoType[],
-  region: string,
-  name: string
-) => {
-  let filteredClusters = clusters;
-
-  if (region)
-    filteredClusters = filteredClusters.filter((c) => c.region === region);
-  if (name)
-    filteredClusters = filteredClusters.filter((c) => c.name.includes(name));
-  return filteredClusters;
+const ClusterRow: React.FC<RowComponentType<ManagedClusterInfoType>> = ({
+  row: cluster,
+}) => {
+  const { t } = useCustomTranslation();
+  const { odfInfo, region, isManagedClusterAvailable } = cluster;
+  const isClientExists = !!odfInfo?.storageClusterInfo?.clientInfo;
+  const odfVersion = odfInfo?.odfVersion;
+  return (
+    <>
+      <Td
+        translate={null}
+        dataLabel={getColumnHelper(COLUMN_NAMES.ManagedCluster, t).columnName}
+      >
+        <Text> {getName(cluster)}</Text>
+      </Td>
+      <Td
+        translate={null}
+        dataLabel={
+          getColumnHelper(COLUMN_NAMES.AvailabilityStatus, t).columnName
+        }
+      >
+        {isManagedClusterAvailable ? (
+          <StatusIconAndText
+            icon={<GreenCheckCircleIcon />}
+            title={t('Online')}
+          />
+        ) : (
+          <StatusIconAndText
+            icon={<RedExclamationCircleIcon />}
+            title={t('Offline')}
+          />
+        )}
+      </Td>
+      <Td
+        translate={null}
+        dataLabel={getColumnHelper(COLUMN_NAMES.DataFoundation, t).columnName}
+      >
+        <Text className={!odfVersion ? 'text-muted' : ''}>
```

Comment:
Optional alternative using `classnames` (up to you):
```suggestion
        <Text className={cn({'text-muted': !odfVersion})}>
```